pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -141,17 +141,39 @@ type Webhook struct {
 	endpoint          string
 	resources         []string
 	operations        []admiv1.OperationType
-	filter            *containers.Filter
+	filters           map[string]*containers.Filter
```

Comment:
I don't think this is a good idea. 

This leads to coupling within the same webhook multiple features that are not actually related.

We should probably have one webhook per feature (or a webhook for a set of features only if it makes sense to, which I assume is a very rare situation)